### PR TITLE
Solution for problem 1

### PR DESCRIPTION
**Pull request by ravic499**

### CODE REVIEW

**Comment by srinathmkce** [comment]
Good Start !!! Keep Going.

**Comment by ravic499** [comment]
Thanks @srinathmkce , Below is the actual problem statement for this solution,

"If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.

Find the sum of all the multiples of 3 or 5 below 1000."

Kindly review my solution and provide me feedback

File: python/problem_1.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def calculate(limit):
+    result = 0
+    for i in xrange(limit):
+        if i % 3 == 0 or i % 5 == 0:
+            result += i
+    print result
```
Comment (diseworks) [inline python/problem_1.py]: Please use python 3

File: python/problem_1.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+def calculate(limit):
+    result = 0
+    for i in xrange(limit):
```
Comment (diseworks) [inline python/problem_1.py]: please use python 3

File: python/problem_1.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+def calculate(limit):
```
Comment (diseworks) [inline python/problem_1.py]: Please add the problem statement

File: python/problem_1.py (hunk 2 above)
Comment (diseworks) [reply python/problem_1.py]: Try to reduce the complexity of this algorithm

**Review by diseworks — CHANGES_REQUESTED** [review]

File: python/problem_1.py (hunk 1 above)
Comment (ravic499) [reply python/problem_1.py]: Thanks. Fixed it.

File: python/problem_1.py (hunk 2 above)
Comment (ravic499) [reply python/problem_1.py]: Can you please be more specific, not able to identify the actual issue

File: python/problem_1.py (hunk 3 above)
Comment (ravic499) [reply python/problem_1.py]: Thanks, Fixed it.

File: python/problem_1.py (hunk 2 above)
Comment (diseworks) [reply python/problem_1.py]: try to find any other way than using for loop

**Review by nirmal040434 — COMMENTED** [review]
Please provide a meaningful file name.

File: python/problem_1.py (hunk 2 above)
Comment (ravic499) [reply python/problem_1.py]: Sorry, I couldn't find any other way.

**Comment by ravic499** [comment]
Thanks @nirmal040434 , I have done the changes

**Review by ravic499 — COMMENTED** [review]
Updated the changes

**Pull request merged**